etc/config-template.toml

```diff
@@ -30,8 +30,12 @@ end-point-concurrency = 8
 # capacity = 0 # 0 is unlimited.
 
 [metric]
-# prometheus client report interval, set 0s to disable prometheus.
-report-interval = "5m"
+# prometheus client push interval, set 0s to disable prometheus from pushing.
+interval = "15s"
+# prometheus pushgateway address, leaves it empty will disable prometheus from pushing.
+address = ""
+# prometheus client push job name. Notice: A node id will automatically appends, e.g., "tikv_1".
```

Comment:
+# the Prometheus client push job name. Note: A node id will automatically append, e.g., "tikv_1".